src/manifest/dbtWorkspaceFolder.ts

```diff
@@ -58,7 +58,7 @@ export class DBTWorkspaceFolder implements Disposable {
         if (j !== i) {
           if (projectFiles[i].fsPath.startsWith(projectFiles[j].fsPath)) {
             window.showWarningMessage(
-              `dbt Power User detected a project located in ${projectFiles[j]} that is contained inside another project ${projectFiles[j]}. This is an unsupported configuration. If you believe this is a valid configuration, please open a github issue.`
+              `dbt Power User detected a project located in ${projectFiles[i]} that is contained inside another project ${projectFiles[j]}. This is an unsupported configuration. If you believe this is a valid configuration, please open a github issue.`
```

Comment:
Please also change both paths to use `.fsPath`, that path will be more meaningful to users.